timemachine/fe/atom_mapping.py

```diff
@@ -115,7 +115,8 @@ def get_cores(
 
     Additional notes
     ----------------
-    1) The returned cores are sorted in increasing order based on the rmsd of the alignment.
+    1) The returned cores are jointly sorted in increasing order based on the number of core-dummy bonds broken, the
+       number of valence values changed and the rmsd of the alignment.
```

Comment:
possible imprecision: "number of valence values changed " sounds like it's the number of core atoms where valence in mol_a differs from valence in mol_b `sum_i (valence_in_b(i) != valence_in_a(i))`, but instead it's something like `sum_i |valence_in_b(i) - valence_in_a(i)|`, not sure how to summarize

```suggestion
    1) The returned cores are jointly sorted in increasing order based on the number of core-dummy bonds broken,
       the sum of valence values changed, and the rmsd of the alignment.
```